### ISSUE TITLE
Modify newscast movies

### ISSUE THREAD

**Issue by peyre** [raceintospace/raceintospace#895]
We've modified the text of some of the newscasts.  For instance:

THE PRIME CONTRACTOR FOR THE CAPSULE PROGRAM ANNOUNCED TODAY THAT THEY CANNOT DELIVER ANY CAPSULES **OR 4-PERSON CAPSULE/MODULES** FOR A YEAR

has been changed to simply
THE PRIME CONTRACTOR FOR THE CAPSULE PROGRAM ANNOUNCED TODAY THAT THEY CANNOT DELIVER ANY CAPSULES FOR A YEAR.

because "4-Person Capsule/Modules" was needlessly confusing.  A new user would see that and think "what's he talking about?"  Of course it refers to direct ascent capsules (highlighting that they also act as lunar landing modules), but the game calls them just "capsules" elsewhere, so calling them "4-person capsule/modules" in needlessly confusing.

It occurred to me today I could probably modify the audio files for those newscasts to match the current text.  At least it should be possible to cut out bits like this.  Adding or modifying would be another story, of course.  I'll have to start keeping track of which ones need fixing.

**Comment by Angel-0a** [comment]
I'd add an even more ambitious goal further down the pipeline - rewriting some of the Russian movies and re-recording them in Russian. IIRC some movies allude to sending people to gulag or for reeducation, etc. in a mocking propaganda style. While this kinda reflects the American view of the USSR at the time and can be rightfully considered to be game's convention and humor, I find it annoying and immersion breaking when playing the Russian side. The space race took place during Khrushchev's era, the worst that could have happened to an underperforming chief engineer was falling out of grace and being relegated to a less prestigious position. 

And of course the newscaster using English with the Russian accent is obviously in a whole another league when it comes to immersion breaking. So does some (if not most) mission movies, where either some very basic Russian is used or again English with the Russian accent. Or no dialogue at all...

Because of this I personally prefer the US side, as it feels more serious and  much more refined multimedia-wise.

**Comment by peyre** [comment]
Valid concerns there.  I complained once about the "managers have been executed" line; it didn't sound right - but Erik Andersen shut me down on that one, insisting that that was really a thing in the 60s.  I couldn't say for sure that he was exaggerating, so I didn't insist.  Modifying Svetlana Izvestia's newscasts would be difficult because it would be hard to simulate her voice though.

However, I don't see why someone couldn't record alternative sounds for some scenes (I'm thinking here of that dopey "Oh, great Russia! We khave landed!")  We could even have someone  (preferably a native speaker) record dialogue in Russian to replace the Russian-accented speech throughout.  Maybe it could be a setting in the config file to play Russian-language tracks for the Soviet player.

**Comment by Angel-0a** [comment]
>Modifying Svetlana Izvestia's newscasts would be difficult because it would be hard to simulate her voice though.

I thought about recording it from scratch in Russian with help of a native speaker but with modern technology maybe some AI tool would suffice?

As for mission movies I think it would be possible to extract some matching audio from clips available on YT. Gagarin's flight with mission control audio is on YT, other activities would probably require some extra digging but maybe it's doable?

**Comment by peyre** [comment]
That might be doable technically, but there could be copyright issues.  Just like my initial plan of grabbing N1, Energia, Lapot, and LK footage from wherever and converting into movies for the game fell apart on copyright grounds.

**Comment by peyre** [comment]
Another newscast that would be good to modify would be the one saying rocket research will be disrupted because a major designer died in surgery (#316).  In fact it affects all research, not just rockets, so the "rocket" bits could be cut out of the audio.

**Comment by peyre** [comment]
Addressing this would also fix #738.

**Comment by peyre** [comment]
Aha, I've found an article on dishonest managers in the time period: [https://warwick.ac.uk/fac/soc/economics/staff/mharrison/public/jce2011postprint.pdf](url).  Here's the money shot:

> Between October 1946 and June 1947, 160 trials took place around the country. Of 249 persons prosecuted between October 1946 and January 1947, 242 were convicted. Offenders were typically given jail time. Four were sentenced to death (with the centre’s approval). A handful received suspended sentences or terms of forced labor with loss of pay at their place of regular employment; the centre criticized these more lenient sentences and appealed them.

That was during Stalin's time too.  Four of 249 were executed - there likely wouldn't have been any under Khrushchev, I'm thinking.

**Comment by peyre** [comment]
Found another one.  In usa_083.ogg, Walcrite accidentally starts to say "this", then cuts to say "next season".  I cut that little bit out, and reduced his emphasis on "next" that he did to cover his error.

**Comment by peyre** [comment]
That should do it.  If anyone finds more changes that should be made to the newscast audio, please let me know - either here, or in another Issue, or by email, whatever.

**Comment by Angel-0a** [comment]
The very first Russian newscast - Izvestia talks vaguely about _the chief designer_, while the text says _designer Player Name_. It would be historically accurate to change the text to match the audio file, as the chief designer's identity was kept secret.

**Comment by peyre** [comment]
That's a good catch.  Korolyov couldn't reveal his own identity even within the Eastern Bloc, and he even kept the location of "Baikonur" secret from his wife.

**Comment by peyre** [comment]
I've just taken care of it.  Now Svetlana doesn't blab your name out on the news for all to hear.

**Comment by Angel-0a** [comment]
Good for her. It would be a pity if she had to be... disciplined.

**Comment by Angel-0a** [comment]
This one has old _executed_ text and new corrected _disciplined_ audio. 

![Image](https://github.com/user-attachments/assets/72a07ce0-9b09-4cd7-8c54-b279349175c5)

**Comment by peyre** [comment]
Yes, I saw that the executed -> disciplined change hadn't been made for some reason so I fixed it.  Sounds like I botched that job!  Now I'm not sure how to fix it.
